api/users/serializers.py

```diff
@@ -87,6 +88,12 @@ class UserSerializer(JSONAPISerializer):
     class Meta:
         type_ = 'users'
 
+    def get_projects_in_common(self, obj):
```

Comment:
Consider adding a `TODO` for a follow-up that returns the users approximately sorted by `projects_in_common`, but that functionality isn't necessary now. See `website.search.elastic_search:search_contributor` for how this could be done, but that's not necessarily an efficient way of doing it.